internal/node/machine_mgr.go

```diff
@@ -352,133 +355,6 @@ func (m *MachineManager) awaitHandshake(vmid string) {
 	}
 }
 
-// Called when the node server gets a log entry via internal NATS. Used to
```

Comment:
I kinda liked having all of the `MachineManager` code in the same place. Maybe it doesn't matter too much. @jordan-rash do you have an opinion?

Reply:
I start to itch when we get over the 500 line mark. While yes it's all under the machine manager umbrella, I like having logically related stuff in nearby locations.

Reply:
> I start to itch when we get over the 500 line mark. While yes it's all under the machine manager umbrella, I like having logically related stuff in nearby locations.

Makes sense. It would be better if we created a new interface and struct to make things more readable imho instead of separating it into separate files. I will not die on this hill though.